### DISCUSSION TITLE
Why Pascal specifically?

### DISCUSSION THREAD

**Discussion by AlpinDale** [red-prig/fpPS4#44] [General]
Any reason why the dev chose Pascal in particular? Or is it just that he's better at it compared to other languages? Just think it's an odd choice.

**Comment by red-prig** [comment]
I want to do it in Free Pascal, I don't care about other people's opinions.

**Reply by gpicc** [reply]
Based

**Reply by DuckersMcQuack** [reply]
As long as it works for you and makes the emulator great nonetheless, and can emulate games in the end, it doesn't really matter what it's written in tbh.

**Comment by gelson-zord** [comment]
I think it's the language the dev dominates more, like the project Heroic Launcher which aims to be a unified alternative launcher for Epic and GOG launchers, which is developed in Electron framework, that the dev dominate and work with.

**Reply by MemoEL7awy1995** [reply]
Please, let us start working. We start working on the program once every month, so that anyone who wants to play PlayStation 4, and because the devices are no longer at a high level. Please, let us work on the program.
